guides/hosting/configurations/shopware/staging.md

```diff
@@ -0,0 +1,186 @@
+---
+nav:
+  title: Staging
+  position: 10
+
+---
+
+# Staging
+
+Since Shopware 6.6.1.0, Shopware has a integrated Staging Mode. This mode prepares the Shop to be used in a staging environment. This means that the shop is prepared to be used in a test environment, where changes can be made without affecting the live shop.
+
+## The workflow
+
+The staging mode is designed, to only modify data inside the Shopware instance. This means, the Staging mode does not duplicate the current Installation, copy Database or Files. It only modifies the data inside the Shopware instance.
+
+So the real world use-case would be kinda like this:
+
+### Creating the second Shopware instance
+
+The recommanded way would to deploy from your Git repository to the new environment. This way, you make sure that the codebase is equal to the live environment. The alternative way would be to copy the files from the live environment to the staging environment.
+
+### Copying the Database
+
+::: info
+You should make sure that the `mysqldump` and the `mysql` binary is from the same major version and vendor. This means, if you use `mysqldump` from MariaDB, you should also use `mysql` from MariaDB. The same applies to MySQL.
+:::
+
+To have the Staging environment similar as possible to the live environment, it's recommanded to duplicate the database. For this you can use the `mysqldump` command to export the database and import it into the staging environment. 
+
+Our recommandation would be to use `shopware-cli project dump` to create a dump of the database and import it with the regular mysql command. Shopware cli does also have a flag to anonymize the data, so you can be sure that no personal data is in the staging environment.
+
+```bash
+# creating a regular dump, the clean parameter will not dump the data of cart table
+shopware-cli project dump --clean --host localhost --username db_user --password db_pass --output shop.sql shopware
+
+# create a dump with anonymized data
+shopware-cli project dump --clean --anonymize --host localhost --username db_user --password db_pass --output shop.sql shopware
+```
+
+You can configure the dump command with a `.shopware-project.yml` which tables should be skipped, which additional fields should be anonymized and more. Checkout the [CLI documentation](https://sw-cli.fos.gg/commands/project/#shopware-cli-project-dump-database) for more information.
+
+### Configuration
+
+::: info
+Generally it's not recommanded to share any resources like MySQL, Redis, ElasticSearch/OpenSearch between the live and staging environment. This could lead to data corruption when the configuration is not done correctly. Also the performance of the live environment could be affected by the staging environment.
```

Comment:
```suggestion
It's not recommanded to share any resources like MySQL, Redis, ElasticSearch/OpenSearch between the live and staging environment. This could lead to data corruption when the configuration is not done correctly. Also, the performance of the live environment could be affected by the staging environment.
```